augly/video/helpers/ffmpeg.py

```diff
@@ -28,11 +29,38 @@ def combine_frames_and_audio_to_file(
             "a directory"
         )
 
-    video_stream = ffmpeg.input(raw_frames, pattern_type="glob", framerate=framerate)
-    video_stream = video_stream.filter(
-        "pad", **{"width": "ceil(iw/2)*2", "height": "ceil(ih/2)*2"}
-    )
-    merge_video_and_audio(video_stream, audio, output_path)
+    temp_video_path = "/tmp/out.mp4"
+    writer = WriteGear(output_filename=raw_frames, logging=True)
+    ffmpeg_command = [
+        "-y",
+        "-framerate",
+        str(framerate),
+        "-pattern_type",
+        "glob",
+        "-i",
+        raw_frames,
+        "-c:v",
+        "libx264",
+        "-pix_fmt",
+        "yuv420p",
+        "-preset",
+        "ultrafast",
+        temp_video_path,
+    ]
+    writer.execute_ffmpeg_cmd(ffmpeg_command)
+    ffmpeg_command = [
+        "-y",
+        "-i",
+        temp_video_path,
+        "-vf",
+        "pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2",
+        "-preset",
+        "ultrafast",
+        temp_video_path,
+    ]
+    writer.execute_ffmpeg_cmd(ffmpeg_command)
+    writer.close()
+    merge_video_and_audio(temp_video_path, audio, output_path)
```

Comment:
```suggestion
    with tempfile.TemporaryDirectory() as tmpdir:
        temp_video_path = os.path.join(tmpdir, "out.mp4")
        writer = WriteGear(output_filename=raw_frames, logging=True)
        ffmpeg_command = [
            "-y",
            "-framerate",
            str(framerate),
            "-pattern_type",
            "glob",
            "-i",
            raw_frames,
            "-c:v",
            "libx264",
            "-pix_fmt",
            "yuv420p",
            "-preset",
            "ultrafast",
            temp_video_path,
        ]
        writer.execute_ffmpeg_cmd(ffmpeg_command)
        ffmpeg_command = [
            "-y",
            "-i",
            temp_video_path,
            "-vf",
            "pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2",
            "-preset",
            "ultrafast",
            temp_video_path,
        ]
        writer.execute_ffmpeg_cmd(ffmpeg_command)
        writer.close()
        merge_video_and_audio(temp_video_path, audio, output_path)
```
It's better practice to use `tempfile` to get a usable temporary path in this case :) (I'm not sure if the `/tmp/` dir exists on every possible platform). This should fix your lint error I think!

Reply:
Also, please add `import tempfile` above on line 8 :)